Gemfile

```diff
@@ -3,6 +3,7 @@ git_source(:github){|repo| "https://github.com/#{repo}.git"}
 
 ruby "2.7.0"
 
+gem "bcrypt", "3.1.13"
```

Comment:
Gem này dùng làm gì?

Reply:
Gem này dùng để mã hoá mật khẩu User bằng thuật toán Bcrypt,  Bcrypt sẽ chuyển mật khẩu thành 1 đoạn mã hash + kèm 1 đoạn mã random ở sau (Salt) để tăng bảo mật cho mật khẩu.